### ISSUE TITLE
create persistent volume usage dashboards

### ISSUE THREAD

**Issue by oke-py** [cloudnativedaysjp/dreamkast-infra#1276]
Dashboards will be helpful for #1265.
Currently, we have PersistentVolumes as below.
Either dashboards per middleware or a dashboard of all PV would be fine.
```
kubectl get pv -o custom-columns='CLAIM:.spec.claimRef.name, CAPACITY:.spec.capacity.storage' --sort-by=.spec.claimRef.name
```
dev
```
CLAIM                                         CAPACITY
elasticsearch-master-elasticsearch-master-0   30Gi
elasticsearch-master-elasticsearch-master-1   30Gi
elasticsearch-master-elasticsearch-master-2   30Gi
mysql-pv-claim                                5Gi
prometheus-k8s-db-prometheus-k8s-0            10Gi
prometheus-k8s-db-prometheus-k8s-1            10Gi
storage-grafana-0                             10Gi
storage-loki-0                                10Gi
storage-loki-0                                10Gi
```
prd
```
CLAIM                                         CAPACITY
elasticsearch-master-elasticsearch-master-0   30Gi
elasticsearch-master-elasticsearch-master-1   30Gi
elasticsearch-master-elasticsearch-master-2   30Gi
prometheus-k8s-db-prometheus-k8s-0            10Gi
prometheus-k8s-db-prometheus-k8s-1            10Gi
storage-grafana-0                             10Gi
storage-loki-0                                10Gi
```

**Comment by inductor** [comment]
Isn't it related to https://github.com/cloudnativedaysjp/dreamkast-infra/issues/1265?

**Comment by oke-py** [comment]
Yes!
> Dashboards will be helpful for #1265.

**Comment by ryojsb** [comment]
こんな感じの認識であってますでしょうか。
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/36046682/144699549-257836f8-edc0-4c17-9b4a-4fe286fb8d50.png">

**Comment by oke-py** [comment]
おー、めっちゃいい感じです！

**Comment by oke-py** [comment]
fixed by #1294 & #1295
